hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/ozoneimpl/ContainerController.java

```diff
@@ -196,6 +196,10 @@ public Iterator<Container<?>> getContainers() {
     return containerSet.getContainerIterator();
   }
 
+  public Iterable<Container<?>> getContainerSet() {
```

Comment:
We actually should remove `ContainerController.getContainers()` and `ContainerSet.getContainerIterator()`.  They do not follow the Java `Iterable` interface convention.  However, I have not changed it here since the change is large.